ctx_test.go

```diff
@@ -3533,9 +3546,9 @@ func Test_Ctx_Send(t *testing.T) {
 	app := New()
 	c := app.AcquireCtx(&fasthttp.RequestCtx{})
 	defer app.ReleaseCtx(c)
-	utils.AssertEqual(t, nil, c.Send([]byte("Hello, World")))
-	utils.AssertEqual(t, nil, c.Send([]byte("Don't crash please")))
-	utils.AssertEqual(t, nil, c.Send([]byte("1337")))
+	utils.AssertEqual(t, nil, c.SendString("Hello, World"))
```

Comment:
This is the test which tested the `Send` method
this should also work with the `Send` method and not with another one